WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterAdapter.kt

```diff
@@ -0,0 +1,26 @@
+package org.wordpress.android.ui.activitylog.list.filter
+
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView.Adapter
+import org.wordpress.android.ui.activitylog.list.filter.ActivityLogTypeFilterViewModel.ItemUiState
+import org.wordpress.android.ui.utils.UiHelpers
+
+class ActivityLogTypeFilterAdapter(private val uiHelpers: UiHelpers) : Adapter<ActivityLogTypeFilterViewHolder>() {
+    private val items = mutableListOf<ItemUiState>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ActivityLogTypeFilterViewHolder {
+        return ActivityLogTypeFilterViewHolder(parent, uiHelpers)
+    }
+
+    override fun getItemCount(): Int = items.size
```

Comment:
Minor: I suggest moving the `getItemCount` below the `onBindViewHolder(...)` function for the below reasons:
1) The `getItemCount` function is on line 7165 of the parent function, while the rest two are on top, which will help preserve the general order. Wdyt?
2) Having `onCreateViewHolder(...)` and `onBindViewHolder(...)` close to each other, and thus grouped, helps with adapter readability. Wdyt?

Reply:
Good call - fixed in 0bb021dd572584d1b349c60bdf00da21cc0a904e